common/djangoapps/student/management/commands/transfer_students.py

```diff
@@ -52,7 +52,6 @@ class Command(TrackedCommand):
                     help="If True, try to transfer certificate items to the new course.")
     )
 
-    @transaction.commit_manually
```

Comment:
Actually I have already replaced `commit_on_success` with `atomic` below. The reason for keeping it inside the loop is that change for each student should get committed as it happens. Otherwise an error would revert all of the work. Or do you think we should be doing that?